New source ReadWn

Nabeelshar opened the pull request:
some issue with plugging ajax chapter

nanihadesuka commented on app/src/main/java/my/noveldokusha/scraper/scraper.kt:
```diff
@@ -26,7 +26,8 @@ object scraper
         BoxNovel(),
-        LightNovelWorld()
+        LightNovelWorld(),
+        Readwn()
```
typo:  Readwn -> ReadWn

nanihadesuka commented on app/src/main/java/my/noveldokusha/scraper/sources/ReadWn.kt:
```diff
@@ -0,0 +1,103 @@
+
+    override suspend fun getChapterList(doc: Document): List<ChapterMetadata>
+    {
+        // val doc = doc.select(".pagination a[href]").attr("href")
```
If not used, delete.

nanihadesuka commented on app/src/main/java/my/noveldokusha/scraper/sources/ReadWn.kt:
```diff
@@ -0,0 +1,103 @@
+        val id = doc.selectFirst(".pagination a[href]")!!.attr( "href")
+        return connect(baseUrl + id)
+            .addHeaderRequest()
+//            .data("novelId", id)
```
Same, delete if not used.

nanihadesuka reviewed (changes requested):
Looks good, do the changes and I merge it.

nanihadesuka commented on app/src/main/java/my/noveldokusha/scraper/sources/ReadWn.kt:
```diff
@@ -37,12 +37,11 @@ class ReadWn : SourceInterface.catalog
 
     override suspend fun getChapterList(doc: Document): List<ChapterMetadata>
```
I get 403 when trying to get catalog or search, the chapters also don't have title. Must be fixed.

Nabeelshar replied:
Don't know how to fix it.

nanihadesuka commented:
@Nabeelshar what/why those images ?

Nabeelshar commented:
> @Nabeelshar what/why those images ?

?? which images?

nanihadesuka commented:
-.- https://github.com/nanihadesuka/NovelDokusha/pull/24/commits/ec086c64693b2f3c43880101f89e1d1eeedfc8e7

Nabeelshar commented:
Sorry about that i deleted two images then i remember i need those for issues but i already deleted them... I am so new to github.
Hopefully you will be understand the request/issue

nanihadesuka commented:
No problem we can rewrite the commits later, but delete all images. Are you using android studio github addon? I recommend it to push your commits to github more easily.

Nabeelshar commented:
No because i messed with code ALOT, so if a apps runs correctly I think that something is wrong with my pc lol.
I redownload and reopen the source code everytime i wanna do something.
